FlowCrypt/Functionality/Mail Provider/Message Provider/ProcessedMessage.swift

```diff
@@ -96,6 +96,7 @@ extension ProcessedMessage {
     ) {
         self.message = message
         (self.text, self.quote) = Self.parseQuote(text: text)
+        self.text = self.text.removingHtmlTags() ?? self.text
```

Comment:
probably we can move `removingHtmlTags` to `parseQuote`, so it'll return text without tags and we won't need to remove them manually each time

Reply:
Sounds good!
I also removed html tags for `quote` too. What do you think?

Reply:
We don't need `?? ""` for quote text, as in this case quote `...` button becomes visible for all messages (even without quotes)

Reply:
Yep!. Missed it